### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by cwq2326**
Added the information for edit, resize and archive feature for the DG.

Also made some minor updates to my PPP, archive and resize command.

### CODE REVIEW

**Review by jiale-c — CHANGES_REQUESTED** [review]
All looks good to me. Prefer if ur edit's sequence diagram can add the call name for the creation of objects EditCommandParser(message) etc. This is better to standardise w the rest of the members!

File: docs/diagrams/EditFeatureActivityDiagram.puml [hunk 1]
```diff
@@ -2,16 +2,20 @@
...
 
-    else ([else])
+    else (            [Else])
+    endif
+    if () then ([Edited field value that needs \nto be unique already \nexists in the address book])
+    : Throws CommandException;
+    else (            [Else])
         :Field values of person in \n the addressbook is updated;
         :Returns Message to user indicating Success;
         :ResultDisplay prints result to user;
```
Comment (DannyDakota) [inline docs/diagrams/EditFeatureActivityDiagram.puml:21]: you are missing an endif resulting in the alternate branches not joining

File: docs/diagrams/EditFeatureSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+AddressBookParser -> EditCommandParser : parse("n/Poppy")
+activate EditCommandParser
+
+create EditPersonDescriptor
+EditCommandParser --> EditPersonDescriptor : EditPersonDescriptor("n/Poppy")
+activate EditPersonDescriptor
+EditPersonDescriptor --> EditCommandParser : d
+deactivate EditPersonDescriptor
+
+EditCommandParser --> EditPersonDescriptor
```
Comment (DannyDakota) [inline docs/diagrams/EditFeatureSequenceDiagram.puml:40]: Missing method name. Should be setName?

File: docs/diagrams/EditFeatureSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+activate EditCommandParser
+
+EditCommandParser --> AddressBookParser
+deactivate EditCommandParser
+
+AddressBookParser -> EditCommandParser : parse("n/Poppy")
+activate EditCommandParser
+
+create EditPersonDescriptor
+EditCommandParser --> EditPersonDescriptor : EditPersonDescriptor("n/Poppy")
```
Comment (DannyDakota) [inline docs/diagrams/EditFeatureSequenceDiagram.puml:35]: The PUML diagram is right for the constructor but i think maybe the updated PNG file hasn't been updated?

**Review by DannyDakota — CHANGES_REQUESTED** [review]
Some minor details to be changed but otherwise Good Job Wen Quan! Thank you for your hard work 💯

File: docs/diagrams/EditFeatureSequenceDiagram.puml (hunk 2 above)
Comment (cwq2326) [reply docs/diagrams/EditFeatureSequenceDiagram.puml]: Thanks for informing me, I have made the changes 💯 .

File: docs/diagrams/EditFeatureSequenceDiagram.puml (hunk 3 above)
Comment (cwq2326) [reply docs/diagrams/EditFeatureSequenceDiagram.puml:35]: Thanks for informing me, I have made the changes 💯 .

File: docs/diagrams/EditFeatureActivityDiagram.puml (hunk 1 above)
Comment (cwq2326) [reply docs/diagrams/EditFeatureActivityDiagram.puml:21]: Thanks for informing me, I have made the changes 💯 .

**Review by jiale-c — APPROVED** [review]
LGTM!

**Review by DannyDakota — APPROVED** [review]
LGTM! 💯

**Pull request merged**